gui.py

```diff
@@ -1,21 +1,26 @@
 #!/usr/bin/env python3
 
-import os
 import json
-import time
-import threading
 import multiprocessing
+import os
+import threading
+import time
+from pathlib import Path
+from turtle import update
+from urllib.request import urlopen
```

Comment:
```suggestion
```

Reply:
Kann raus - hatte davor die Prüfung in versch. Files bevor ich es als funktion in utils umgesetzt habe